### ISSUE TITLE
Switch names indistinguishable from category members

### ISSUE THREAD

**Issue by Jedsters** [FrSkyRC/ETHOS-Feedback-Community#1292]
If a LSW is created either intentionally or accidentally with the same name as a Category Member (RSSI, FSW1, Throttle Hold etc. etc.) then on the LSW edit screen it is impossible to determine if the Value is a Member or another  LSW name. 
e.g. create a LSW with a name of RSSI
create another LSW using AND and select LSW RSSI for Value 1 and Telemetry RSSI for Value 2
On the LSW edit screen you now can't tell them apart.

Granted, it isn't ideal to create a LSW with the same name as a category member, but these things can happen by accident if we aren't familiar with the name of all members. 

Maybe LSW names could be distinguished from members somehow?
Random thought, Member names could be indicated as _RSSI_   it would have to be something which we can't enter ourselves (so my initial idea of <RSSI> wouldn't be any good as we can type that!)

**Comment by bsongis-frsky** [comment]
You may change all names, I am not sure that the software is able to fix all user mistakes!

**Comment by Jedsters** [comment]
I agree that you can't stop us duplicating a name, however indicating system names e.g. in italic would help us when we do get things wrong by accident.

**Comment by bsongis-frsky** [comment]
And what is a system name? RSSI is not a system name

**Comment by Jedsters** [comment]
Apologies, I was being lazy with my terminology. What I really mean is that there should be a way to distinguish a user assigned name from another name with perhaps one or the other being in italic. (OK, there are other user assigned names too, but as far as I can think, there is only any ambiguity in the case of LSW names with names used in other Categories).
I suspect this would need to be applied anywhere a logic switch name can be used (including Flight Modes, Mixers, Timers, Logic Switches, Special Funcs). 
Given how wide this change would be, maybe there is an easier way of just using an italic case (or similar) for LSW names everywhere they are displayed?

**Comment by agibson2** [comment]
I think the point bsongis is making is RSSI is a default name used for RSSI sensor name but it is just a suggestion and not hard coded to be RSSI.  The default can be changed to anything.
If the user sees a duplicate, they can freely change their RSSI LSW to LSRSSI.
The only way I can think of to not confuse anything would be for every listing in the selection dialog to have another column beside it saying what kind of thing it is (LSW, SENSOR, etc).  Since the user is in control of what they label things though, they could easily name things to not get confused.

**Comment by Jedsters** [comment]
Thanks @agibson2 but I'm not sure that changes anything. If someone renamed their RSSI as Signal and then created a LSW called Signal there would still be a problem so I'm thinking that simply by indicating all LSW names in e.g. italic wherever they are used gets around the problem whatever people call telemetry and LSW. You can then always tell if something is a LSW or it's other instance.

**Comment by mawzthefinn** [comment]
Perhaps a letter prefix indicating where the name is sourced from?

OTX does this with Inputs to distinguish them from raw sticks.

**Comment by Jedsters** [comment]
Bump

**Comment by bsongis-frsky** [comment]
A small icon will be added in 1.7.0 near all sources / switches

**Comment by bsongis-frsky** [comment]
Implemented since a few weeks and available in the current 1.7.0 nightlies
